docker/testing/ubuntu2204_java_driver_testing.docker

```diff
@@ -53,4 +53,12 @@ RUN git clone https://github.com/riptano/ccm.git  && \
     cd ccm && \
     pip install -e .
 
+# download cassandra binaries
+RUN ccm create download -v 3.11.17 && \
+	ccm remove && \
+	ccm create download -v 4.0.14 && \
+	ccm remove && \
+	ccm create download -v 5.0.2 && \
+	ccm remove
```

Comment:
be aware these will quickly be out of date (not the latest patch) versions.

it should be possible to find these values dynamically via https://downloads.apache.org/cassandra/
(that would mean you only have to rebuild and publish after each new patch version, rather than edit this file)

Reply:
Do you happen to know an existing snippet parsing those version strings and output the latest versions?
I would love to update the `Jenkins-asf` file to figure out the latest version in the same way, too.

Reply:
We do something like this for the internal Jenkins runners @SiyaoIsHiding ... check with me tomorrow and I can point you to the right place.

Reply:
there's kinda an example here: https://github.com/apache/cassandra/blob/trunk/.build/docker/ubuntu2004_test.docker#L129 

(but sounds like what @absurdfarce has is more applicable, and tested)

Reply:
Thanks! They are both helpful